### ISSUE TITLE
Odoo.sh web_studio Issue when language is not English

### ISSUE THREAD

**Issue by qndi** [odoo/odoo#31588]
Impacted versions:
Odoo.sh 12.0

Steps to reproduce:
1. Create a new Database (or Rebuild Develop-branch with no modules)
2. Install web_studio
3. load language "de_DE" (German/Deutsch)
4. set language of admin-user to "de_DE"
5. Change any view with Studio (e.g. add field to tree-view of res.users)

Current behavior:
Error: 
Odoo-Server Error (see screenshot)

Expected behavior:
Works

Video/Screenshot link (optional):
![image](https://user-images.githubusercontent.com/5363782/53800932-2b63e500-3f3e-11e9-853c-a8d25d7ccc00.png)

Web-Console Log:
erver application error {"code":200,"message":"Odoo Server Error","data":{"name":"odoo.exceptions.ValidationError","debug":"Traceback (most recent call last):\n  File \"/home/odoo/src/odoo/odoo/models.py\", line 1108, in _validate_fields\n    check(self)\n  File \"/home/odoo/src/odoo/odoo/addons/base/models/ir_ui_view.py\", line 343, in _check_xml\n    view_arch = etree.fromstring(view.arch.encode('utf-8'))\n  File \"src/lxml/etree.pyx\", line 3213, in lxml.etree.fromstring\n  File \"src/lxml/parser.pxi\", line 1876, in lxml.etree._parseMemoryDocument\n  File \"src/lxml/parser.pxi\", line 1764, in lxml.etree._parseDoc\n  File \"src/lxml/parser.pxi\", line 1126, in lxml.etree._BaseParser._parseDoc\n  File \"src/lxml/parser.pxi\", line 600, in lxml.etree._ParserContext._handleParseResultDoc\n  File \"src/lxml/parser.pxi\", line 710, in lxml.etree._handleParseResult\n  File \"src/lxml/parser.pxi\", line 650, in lxml.etree._raiseParseError\n  File \"<string>\", line 0\nlxml.etree.XMLSyntaxError: <no detail available>\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/odoo/src/odoo/odoo/http.py\", line 654, in _handle_exception\n    return super(JsonRequest, self)._handle_exception(exception)\n  File \"/home/odoo/src/odoo/odoo/http.py\", line 312, in _handle_exception\n    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])\n  File \"/home/odoo/src/odoo/odoo/tools/pycompat.py\", line 87, in reraise\n    raise value\n  File \"/home/odoo/src/odoo/odoo/http.py\", line 696, in dispatch\n    result = self._call_function(**self.params)\n  File \"/home/odoo/src/odoo/odoo/http.py\", line 344, in _call_function\n    return checked_call(self.db, *args, **kwargs)\n  File \"/home/odoo/src/odoo/odoo/service/model.py\", line 97, in wrapper\n    return f(dbname, *args, **kwargs)\n  File \"/home/odoo/src/odoo/odoo/http.py\", line 337, in checked_call\n    result = self.endpoint(*a, **kw)\n  File \"/home/odoo/src/odoo/odoo/http.py\", line 939, in __call__\n    return self.method(*args, **kw)\n  File \"/home/odoo/src/odoo/odoo/http.py\", line 517, in response_wrap\n    response = f(*args, **kw)\n  File \"/home/odoo/src/enterprise/web_studio/controllers/main.py\", line 492, in edit_view\n    self._set_studio_view(view, new_arch)\n  File \"/home/odoo/src/enterprise/web_studio/controllers/main.py\", line 391, in _set_studio_view\n    self._create_studio_view(view, arch)\n  File \"/home/odoo/src/enterprise/web_studio/controllers/main.py\", line 542, in _create_studio_view\n    'name': self._generate_studio_view_name(view),\n  File \"<decorator-gen-123>\", line 2, in create\n  File \"/home/odoo/src/odoo/odoo/api.py\", line 430, in _model_create_single\n    return create(self, arg)\n  File \"/home/odoo/src/enterprise/web_studio/models/studio_mixin.py\", line 19, in create\n    res = super(StudioMixin, self).create(vals)\n  File \"<decorator-gen-32>\", line 2, in create\n  File \"/home/odoo/src/odoo/odoo/api.py\", line 451, in _model_create_multi\n    return create(self, [arg])\n  File \"/home/odoo/src/odoo/odoo/addons/base/models/ir_ui_view.py\", line 428, in create\n    return super(View, self).create(vals_list)\n  File \"<decorator-gen-3>\", line 2, in create\n  File \"/home/odoo/src/odoo/odoo/api.py\", line 452, in _model_create_multi\n    return create(self, arg)\n  File \"/home/odoo/src/odoo/odoo/models.py\", line 3569, in create\n    fields[0].determine_inverse(batch_recs)\n  File \"/home/odoo/src/odoo/odoo/fields.py\", line 1109, in determine_inverse\n    getattr(records, self.inverse)()\n  File \"/home/odoo/src/odoo/odoo/addons/base/models/ir_ui_view.py\", line 275, in _inverse_arch\n    view.write(data)\n  File \"/home/odoo/src/enterprise/web_studio/models/studio_mixin.py\", line 28, in write\n    res = super(StudioMixin, self).write(vals)\n  File \"/home/odoo/src/odoo/odoo/addons/base/models/ir_ui_view.py\", line 444, in write\n    return super(View, self).write(self._compute_defaults(vals))\n  File \"/home/odoo/src/odoo/odoo/models.py\", line 3284, in write\n    self._write(store_vals)\n  File \"/home/odoo/src/odoo/odoo/models.py\", line 3430, in _write\n    self._validate_fields(vals)\n  File \"/home/odoo/src/odoo/odoo/models.py\", line 1112, in _validate_fields\n    raise ValidationError(\"%s\\n\\n%s\" % (_(\"Error while validating constraint\"), tools.ustr(e)))\nodoo.exceptions.ValidationError: ('Fehler während der Constraint-Validierung\\n\\nNone', None)\n","message":"Fehler während der Constraint-Validierung\n\nNone\nNone","arguments":["Fehler während der Constraint-Validierung\n\nNone",null],"exception_type":"validation_error"}}

With a local environment I was not able to reproduce the error.

**Comment by pedrobaeza** [comment]
@mart-e is #31585 for solving this?

**Comment by mart-e** [comment]
@pedrobaeza yes (but by the way, you shouldn't be modifying your views in another language than English)

**Comment by qndi** [comment]
I have now updated my local 18-day-old Odoo-Code and now I am able to reproduce the error.
So it is very likely related to #31585 .

**Comment by pedrobaeza** [comment]
Already merged.

**Comment by datenpol** [comment]
This merge was only for 11.0. Will there be a separate merge for 12.0?

**Comment by pedrobaeza** [comment]
It's forward-ported during the week.
